doc/Command-Reference.md

```diff
@@ -5603,5 +5608,76 @@ This command displays the routing policy that takes precedence over the other ro
     Action:
       Exit routemap
   ```
-
 Go Back To [Beginning of the document](#) or [Beginning of this section](#quagga-bgp-show-commands)
+
+## Container Auto-restart
```

Comment:
Please move this entire section so that it is in alphabetical order (between the BGP and DHCP relay commands).

Reply:
Fixed.